sklearn/linear_model/bayes.py

```diff
@@ -323,6 +371,18 @@ class ARDRegression(LinearModel, RegressorMixin):
     Notes
     --------
     See examples/linear_model/plot_ard.py for an example.
+
+    References
+    -------
+    D. J. C. MacKay, Bayesian nonlinear modeling for the prediction competition,
+    ASHRAE Transactions, 1994.
+
+    See also: http://www.utstat.toronto.edu/~rsalakhu/sta4273/notes/Lecture2.pdf
```

Comment:
Please use `http://www.utstat.toronto.edu/~rsalakhu/sta4273/notes/Lecture2.pdf#page=15`

Perhaps format as link with anchor "Lecture slides on ..." and cite "Russ Salakhutdinov" by name.